examples/MatLabAgent_v2/setup.py

```diff
@@ -0,0 +1,26 @@
+from setuptools import setup, find_packages
+
+MAIN_MODULE = 'agent'
+
+# Find the agent package that contains the main module
+packages = find_packages('.')
+agent_package = 'mlaV2'
```

Comment:
This is another place where the auto generated package name could use updation. mlaV2 sounds cryptic.